packages/pharos/src/components/checkbox-group/pharos-checkbox-group.ts

```diff
@@ -81,16 +81,11 @@ export class PharosCheckboxGroup extends FormElement {
   }
 
   protected override render(): TemplateResult {
-    const labels = ['legend'];
-    if (this.messageId) {
-      labels.push(this.messageId);
-    }
-
     return html`
       <fieldset
         class="checkbox-group checkbox-group--${this.horizontal ? 'horizontal' : 'vertical'}"
         aria-invalid="${this.invalidated}"
-        aria-labelledby="${labels.join(' ')}"
+        aria-describedby="${this.messageId}"
```

Comment:
You'll need to wrap this in the `ifdefined` Lit directive, as `this.messageId` can be `undefined` and we don't want to render `aria-describedby="undefined"` literally.